api/python/quilt3/packages.py

```diff
@@ -1391,12 +1396,34 @@ def dest_fn(lk, *args, **kwargs):
         registry = get_package_registry(registry)
         self._validate_with_workflow(registry=registry, workflow=workflow, name=name, message=message)
 
+        def check_latest_hash():
+            if force:
+                return
+
+            try:
+                latest_hash = get_bytes(registry.pointer_latest_pk(name)).decode()
+            except botocore.exceptions.ClientError as ex:
+                if ex.response['Error']['Code'] == 'NoSuchKey':
+                    # Expected
+                    return
+                raise
+
+            if self._origin is None or latest_hash != self._origin.top_hash:
+                raise QuiltException(
+                    f"Package with an unexpected hash {latest_hash} already exists at the destination; "
```

Comment:
Might make sense to specify both expected and actual hashes.

Reply:
Ok